camino-messenger-bot-distributor.yaml.example

```diff
@@ -66,10 +59,11 @@ rpc_url: wss://columbus.camino.network/ext/bc/C/ws
 ### Booking Token
 
 # Booking Token contract address. This address will be used for minting, buying of Booking Tokens
-booking_token_address: "0xd4e2D76E656b5060F6f43317E8d89ea81eb5fF8D"
+booking_token_address: "0x1B9dac1f224DAEA0bB1Ee5e01Eb51E4551CEb7AB"
```

Comment:
This is not the address of the BookingToken. I think you put your test CMAccount address here.